gandicli.man.rst

```diff
@@ -98,6 +98,9 @@ Namespaces:
 *  dns keys recover        Recover deleted key for a domain.
 *  dns list                Display records for a domain.
 *  dns update              Update record entry for a domain.
+*  dnssec create           Create DNSSEC key.
```

Comment:
@grigouze please add dnssec methods documentation with parameters list below in the gandicli.man.rst (check for `gandi dns update`).

Reply:
@aegiap done